pages/agent/v3/ssh_keys.md.erb

```diff
@@ -123,3 +123,19 @@ set -eu
 
 export SSH_AUTH_SOCK="/var/lib/buildkite-agent/.ssh/ssh-agent.sock"
 ```
+
+## Using multiple keys by adding pipeline-specific environments  
+
+There is yet another way to use multiple SSH keys.  
+
+1. Add a pipeline-specific environment (for example, by using [Elastic Stack's secrets support](https://github.com/buildkite/elastic-ci-stack-for-aws#build-secrets) or by having an Agent environment hook that switches on the repository URL or the pipeline slug):  
```

Comment:
Do we mention this in the docs and can we link there instead of the GitHub repo?